### ISSUE TITLE
Is support compontent tag ?

### ISSUE THREAD

**Issue by MilesWuCode** [nuxt-community/apollo-module#374]
Is support compontent tag ?
or
can i use webpack setup

vue-apollo
https://apollo.vuejs.org/guide/components/query.html#tag-setup

**Comment by ehsanonline** [comment]
nope, there is a [todo](https://github.com/nuxt-modules/apollo/blob/c93fbf0b0af2b0fcfe241d67669ad72c70d7251d/src/module.ts#L118) though
